### ISSUE TITLE
mkvinfo gui window black and unresponsive during start with filename

### ISSUE THREAD

**Issue by smidm** [mbunkus/mkvtoolnix#1815]
# To Reproduce

    $ mkvinfo --gui filename.mkv

Displays black window unresponsive for tenths of seconds. I work with big (1-8 GB) files on an external drive.

# Expected

The gui should not wait on file load during startup and rather initialize the main window and display a progress bar.

# Environment

os: linux, fedora 23 

    $ mkvinfo --version
    mkvinfo v9.5.0 ('Quiet Fire') 64bit

**Comment by mbunkus** [comment]
mkvinfo's GUI is not multi-threaded. It's barely bolted onto the CLI part. I have no plans for changing this. What I do plan on doing is rewriting it one day to fit into MKVToolNix GUI. During such a rewrite multi-threading will be introduced, too.

However, I cannot give you an ETA on when I will get around to it.

I will therefore close this issue as "won't fix" for the time being.

**Comment by smidm** [comment]
In that case at least a message in terminal would help a lot. When I encountered the black unresponsive window I wasn't aware that the thing works.
